lib/rest-api.php

```diff
@@ -183,3 +183,38 @@ function gutenberg_auto_draft_get_sample_permalink( $permalink, $id, $title, $na
 	return $permalink;
 }
 add_filter( 'get_sample_permalink', 'gutenberg_auto_draft_get_sample_permalink', 10, 5 );
+
+/**
+ * Filters WP_User_Query arguments when querying users via the REST API.
+ *
+ * Allow who=has_published_post.
+ *
+ * @param array           $prepared_args Array of arguments for WP_User_Query.
+ * @param WP_REST_Request $request       The REST API request.
+ *
+ * @return array Returns modified $prepared_args.
+ */
+function gutenberg_rest_user_query_has_published_posts( $prepared_args, $request ) {
+	error_log( print_r( $prepared_args, true ) );
```

Comment:
```suggestion
```

Reply:
:laughing: yeah, I missed this in the initial commit. Pushed a fix for this already in https://github.com/WordPress/gutenberg/pull/32662/commits/9b33ffbde33c9cff5b5e4c7dfe761055caa713cf :+1: